Sphinx moved away from docutils.utils.roman to a separate package because
the distutils one is being removed. Our version of distutils still has
that functionality, so let's use that one.

Not intended for upstream.

--- sphinx-8.2.0/PKG-INFO
+++ sphinx-8.2.0/PKG-INFO
@@ -56,7 +56,6 @@
 Requires-Dist: babel>=2.13
 Requires-Dist: alabaster>=0.7.14
 Requires-Dist: requests>=2.30.0
-Requires-Dist: roman-numerals-py>=1.0.0
 Requires-Dist: packaging>=23.0
 Requires-Dist: colorama>=0.4.6; sys_platform == 'win32'
 Requires-Dist: sphinxcontrib-websupport ; extra == "docs"
--- sphinx-8.2.0/pyproject.toml
+++ sphinx-8.2.0/pyproject.toml
@@ -71,7 +71,6 @@
     "babel>=2.13",
     "alabaster>=0.7.14",
     "requests>=2.30.0",
-    "roman-numerals-py>=1.0.0",
     "packaging>=23.0",
     "colorama>=0.4.6; sys_platform == 'win32'",
 ]
--- sphinx-8.2.0/sphinx/writers/latex.py
+++ sphinx-8.2.0/sphinx/writers/latex.py
@@ -12,7 +12,6 @@ from pathlib import Path
 from typing import TYPE_CHECKING, cast
 
 from docutils import nodes, writers
-from roman_numerals import RomanNumeral
 
 from sphinx import addnodes, highlighting
 from sphinx.errors import SphinxError
@@ -24,6 +23,8 @@ from sphinx.util.nodes import clean_aste
 from sphinx.util.template import LaTeXRenderer
 from sphinx.util.texescape import tex_replace_map
 
+from docutils.utils.roman import toRoman
+
 if TYPE_CHECKING:
     from collections.abc import Iterable
     from typing import Any, ClassVar
@@ -1425,8 +1426,8 @@ class LaTeXTranslator(SphinxTranslator):
                 return get_nested_level(node.parent)
 
         nested_level = get_nested_level(node)
-        enum = f'enum{RomanNumeral(nested_level).to_lowercase()}'
-        enumnext = f'enum{RomanNumeral(nested_level + 1).to_lowercase()}'
+        enum = f'enum{toRoman(nested_level).lower()}'
+        enumnext = f'enum{toRoman(nested_level + 1).lower()}'
         style = ENUMERATE_LIST_STYLE.get(get_enumtype(node))
         prefix = node.get('prefix', '')
         suffix = node.get('suffix', '.')
